lib/jsdom/level2/html.js

```diff
@@ -1270,6 +1270,14 @@ define('HTMLLegendElement', {
   proto: {
     get form() {
       return closest(this, 'FORM');
+    },
+    click: function() {
```

Comment:
Why is this defined on HTMLLegendElement? You should be modifying the `click()` method of HTMLElement.